geoh5py/objects/surveys/direct_current.py

```diff
@@ -0,0 +1,282 @@
+#  Copyright (c) 2021 Mira Geoscience Ltd.
+#
+#  This file is part of geoh5py.
+#
+#  geoh5py is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  geoh5py is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public License
+#  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+import uuid
+
+import numpy as np
+
+from ...data import Data, ReferencedData
+from ..curve import Curve
+from ..object_type import ObjectType
+
+
+class PotentialElectrode(Curve):
+    """
+    Ground potential electrode (receiver).
+
+    .. warning:: Partially implemented.
+
+    """
+
+    __TYPE_UID = uuid.UUID("{275ecee9-9c24-4378-bf94-65f3c5fbe163}")
+
+    def __init__(self, object_type: ObjectType, **kwargs):
```

Comment:
There's no way to know from the docs or the init signature what the attributes for this class are (downside of opening up kwargs).  I would recommend documenting, or replacing the kwargs with a list of arguments if it's short enough, or both!  Another option is to set all the valid attributes for the class to None in the init and then update with kwargs.

Reply:
...this would allow you to type the arguments too!

Reply:
isn't implied with the base class though?